test/webgl/context-state.spec.js

```diff
@@ -2,8 +2,8 @@ import {GL} from 'luma.gl';
 import {getParameter, getParameters, setParameters, withParameters, resetParameters} from 'luma.gl';
 import {GL_PARAMETER_DEFAULTS as GL_PARAMETERS} from '../../src/webgl-utils/parameter-access';
 import {
-  GL_PARAMETER_SETTINGS_TWO,
-  GL_PARAMETER_SETTINGS_TWO_ENUM_FUNCTION
+  GL_PARAMETER_SETTINGS_ONE,
```

Comment:
Maybe call these constants `PARAMETER_STYLE_SETTINGS` and `FUNCTION_STYLE_SETTINGS`?

We may also need to describe these concepts in the docs, so good to agree on a terminology.